var/spack/repos/builtin/packages/h5bench/package.py

```diff
@@ -58,3 +59,33 @@ def cmake_args(self):
         ]
 
         return args
+
+    @run_after("install")
+    def setup_build_tests(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources(["tests", "samples"])
+
+        launcher = self.mpi_launcher()
+
+        filter_file(r"mpirun", launcher.command, "samples/sync-write-1d-contig-contig.json")
+
+    def mpi_launcher(self):
+        searchpath = [self.spec["mpi"].prefix.bin]
+        try:
+            searchpath.insert(0, self.spec["slurm"].prefix.bin)
+        except KeyError:
+            print("Slurm not found, ignoring.")
+        commands = ["srun", "mpirun", "mpiexec"]
+        return which(*commands, path=searchpath) or which(*commands)
+
+    def test_help(self):
+        """Stand-alone/smoke test."""
```

Comment:
The docstring becomes the description of the test part.  Perhaps something along the following lines is more descriptive:

```suggestion
        """run h5bench help"""
```

Reply:
done!